### DISCUSSION TITLE
ERROR: ld.so: object '/milvus/lib/' from LD_PRELOAD cannot be preloaded (cannot read file data): ignored.

### DISCUSSION THREAD

**Discussion by rere950303** [milvus-io/milvus#38702] [Q&A and General discussion] [answered]
- version: v2.4.18
- mode: distributed

When deploying Milvus, each component is showing the following error logs. 

```
ERROR: ld.so: object '/milvus/lib/' from LD_PRELOAD cannot be preloaded (cannot read file data): ignored.
ERROR: ld.so: object '/milvus/lib/' from LD_PRELOAD cannot be preloaded (cannot read file data): ignored.
ERROR: ld.so: object '/milvus/lib/' from LD_PRELOAD cannot be preloaded (cannot read file data): ignored.
```

When running `echo $LD_PRELOAD`, it shows `/milvus/lib/:/milvus/lib/libjemalloc.so`. 
Should we change the `LD_PRELOAD` environment variable value from `/milvus/lib/:/milvus/lib/libjemalloc.so` to `/milvus/lib/libjemalloc.so`?

**Comment by rere950303** [comment]
@yhmo could you check this issue plz?? 🙏

**Comment by yhmo** [comment, marked as answer]
The same issue: https://github.com/milvus-io/milvus/issues/38468
I suppose this error can be ignored.

**Comment by xiaofan-luan** [comment]
@yhmo 
can we remove this error message to avoid confusing?
